### DISCUSSION TITLE
Anything I can do to help move PR's on?

### DISCUSSION THREAD

**Discussion by cmcnicholas** [getlift/lift#374] [General]
There are a number of quality of life and feature PR's open for some time, just wondering if there is anything I can do to help move these forward, I'm running a separate fork atm and publishing separately to get some of these in.

**Comment by mnapoli** [comment]
Personally I have limited time to review PRs lately. And the hardest part with PRs is maintaining the code provided by others, for use cases we don't always fully understand, for the next X years 😅 It's not that I don't want to do it, it's that I have to choose where to invest my OS time carefully ^^
